crda/remotematcher.go

```diff
@@ -0,0 +1,258 @@
+package crda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"golang.org/x/sync/errgroup"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+var (
+	_ driver.Matcher       = (*Matcher)(nil)
+	_ driver.RemoteMatcher = (*Matcher)(nil)
+)
+
+const (
+	// Bounded concurrency limit.
+	defaultRequestConcurrency = 10
```

Comment:
this this be based on cpu count ? 

```
runtime.NumCPU()
```

Reply:
@ldelossa This bounded concurrency number is for http requests. Since it is an I/O bound operation, IMHO it may not be helpful to tying it with `runtime.NumCPU()`.

Reply:
I see, you want to get them all in flight first, and then let the scheduler park them until io is ready? I could see the logic in that.